archive/zip_archiver.go

```diff
@@ -71,17 +73,33 @@ func (a *ZipArchiver) ArchiveFile(infilename string) error {
 	return err
 }
 
-func checkMatch(fileName string, excludes []string) (value bool) {
+func checkMatch(fileName, indirname string, excludes []string) (value bool, err error) {
```

Comment:
shouldn't this be
func checkMatch(fileName string, indirname string, excludes []string) (value bool, err error) {

Reply:
This is valid Golang I believe. Does this project prefer to explicitly specify variable types?